R/drive_cp.R

```diff
@@ -47,35 +49,39 @@ drive_cp <- function(file, path = NULL, name = NULL, verbose = TRUE) {
   }
 
   if (is_path(path)) {
-    if (is.null(name) && drive_path_exists(append_slash(path))) {
-      path <- append_slash(path)
-    }
+    confirm_clear_path(path, name)
```

Comment:
I don't love this name, but neither can I suggest a better one right now. It's internal anyway so 👍